gui.py

```diff
@@ -5,56 +5,84 @@
 import PySimpleGUI as sg
 
 from db import store_translation, retrieve_translations
-from models import Translation
 from translate import translate_text
 
+import pyperclip
+import math
```

Comment:
PEP8 suggests Standard Library - 3rd party - own, but ok for now, let me add an issue to add pre-commit + isort